jerry-core/vm/opcodes.c

```diff
@@ -153,7 +153,23 @@ opfunc_set_accessor (bool is_getter, /**< is getter accessor */
                      ecma_value_t accessor) /**< accessor value */
 {
   ecma_object_t *object_p = ecma_get_object_from_value (object);
-  ecma_string_t *accessor_name_p = ecma_get_string_from_value (accessor_name);
+  ecma_string_t *accessor_name_p;
+  bool need_cleanup = false;
+  if ((accessor_name & ECMA_VALUE_TYPE_MASK) == ECMA_TYPE_STRING)
```

Comment:
Please use the helper macros from ecma-helpers.h. Also I think we can simply use the to_string in the non-string case, since that is very rare. We can use an assert since the conversion must be successful (in es5.1 at least).